src/webapp/pages/settings/SettingsPage.tsx

```diff
@@ -20,15 +19,19 @@ import { PageHeader } from "../../components/page-header/PageHeader";
 import { PermissionsDialog, SharedUpdate } from "../../components/permissions-dialog/PermissionsDialog";
 import { useAppContext } from "../../contexts/app-context";
 import { DhisPage } from "../dhis/DhisPage";
+import { CustomizeSettingsDialog } from "../../components/customize-settings-dialog/CustomizeSettingsDialog";
+import { useAppConfigContext } from "../../contexts/AppConfigProvider";
 
 export const SettingsPage: React.FC = () => {
-    const { modules, landings, reload, usecases, setAppState, showAllModules, isLoading, isAdmin } = useAppContext();
+    const { modules, landings, reload, usecases, setAppState, isLoading, isAdmin } = useAppContext();
+
+    const { appConfig, logoInfo, save, hasLoaded } = useAppConfigContext();
 
     const snackbar = useSnackbar();
     const loading = useLoading();
 
     const [permissionsType, setPermissionsType] = useState<string | null>(null);
-    const [settingsPermissions, setSettingsPermissions] = useState<Permission>();
+    const [showCustomSettings, setShowCustomSettings] = useState<boolean>(false);
```

Comment:
There is no need to add the type for the `useState` as it will be caught from the `false`